block_opencast.php

```diff
@@ -59,12 +59,27 @@ public function get_content() {
         }
 
         $renderer = $PAGE->get_renderer('block_opencast');
-        $apibridge = \block_opencast\local\apibridge::get_instance();
-        try {
-            $videos = $apibridge->get_block_videos($COURSE->id);
-        } catch (\moodle_exception $e) {
-            $videos = new \stdClass();
-            $videos->error = $e->getmessage();
+
+        $cache = cache::make('block_opencast', 'videodata');
+        if ($cache->has($COURSE->id)) {
```

Comment:
`get()` already returns `false` if the key is not set, so the double hit is unnecessary https://docs.moodle.org/dev/Cache_API_-_Quick_reference#Get_a_key